database/sql.php

```diff
@@ -10,9 +10,9 @@ public function makeValue($queryValue) {
         if ($queryValue->plain) {
             $str = $queryValue->value;
         } else {
-            $str = jsenc($queryValue->value);
+            $str = json_encode($queryValue->value);
             if (is_array($queryValue->value)) {
-                $str = jsenc($str);
+                $str = json_encode($str);
```

Comment:
Ich weiß: Hier hat sich das Verhalten nicht geändert. Aber ist das wirklich sinnvoll hier doppelt zu encodieren?

Reply:
@bensteffen ?

Reply:
Das ist, wenn man in einer Entität ein Feld mit Typ "object" verwendet und man vergisst das Objekt vorher zu Stringifien, wie es hier in AbstractSqlConnection immer passiert:
![image](https://user-images.githubusercontent.com/37374938/62480007-03a7d000-b7af-11e9-933d-cd17f8d22559.png)

JSON-Strings müssen immer zweimal encodiert werden, damit man die Gänsefüßchen nicht verliert beim Parsen. Ist also hier quasi nur ein zweites Netz.

Reply:
Das hat was mit dem Feld-Typ "object" und dem damit verbundenen ablegen von JSON-Strings zu tun. Ich teste nochmal, ob das wirklich nötig ist.